Makefile

```diff
@@ -73,11 +75,20 @@ setup-auth:
 	@sed -i "" 's/Y2xvdWQucmVkaGF0LmNvbSB1c2VybmFtZQ==/$(shell printf "$(shell echo $(or $(USER),cloud.redhat.com username))" | base64)/g' testing/authentication_secret.yaml
 	@sed -i "" 's/Y2xvdWQucmVkaGF0LmNvbSBwYXNzd29yZA==/$(shell printf "$(shell echo $(or $(PASS),cloud.redhat.com password))" | base64)/g' testing/authentication_secret.yaml
 
+add-prom-route:
+	@echo 'spec:' >> testing/cost-mgmt_v1alpha1_costmanagement.yaml
+	@echo '  prometheus_config:' >> testing/cost-mgmt_v1alpha1_costmanagement.yaml
+	@echo '    service_address: $(EXTERNAL_PROM_ROUTE)'  >> testing/cost-mgmt_v1alpha1_costmanagement.yaml
+	@echo '    skip_tls_verification: true' >> testing/cost-mgmt_v1alpha1_costmanagement.yaml
+
 deploy-cr:
 	@cp config/samples/cost-mgmt_v1alpha1_costmanagement.yaml testing/cost-mgmt_v1alpha1_costmanagement.yaml
+	$(MAKE) add-prom-route
```

Comment:
can we add a variable here for whether or not we should add the external route to the spec? Kind of the same way that we check for auth below. I use the `deploy-cr` command whether I'm running as a pod inside the cluster or on my local machine.